reactApp2/list.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FilteredList from './filteredList.jsx';
+
+
+class List extends React.Component { 
+  constructor(props) {
+    super(props);
+    this.state = {nameList: [], selectedFilter: ""};
+    this.handleClick = this.handleClick.bind(this);
+    this.handleActiveClick = this.handleActiveClick.bind(this);
+    this.handleDeletedClick = this.handleDeletedClick.bind(this);
+    this.handleAllClick = this.handleAllClick.bind(this);
+  }
+  
+  
+  componentWillReceiveProps(newProps) { 
+    let nameList = this.state.nameList.slice();
+    let object = {name: newProps.name, isDeleted: false, Color: ''};
+    if (!nameList.find((element) => (object.name == "" || element.name == object.name)))
+      nameList.push(object);
+    this.setState({ nameList: nameList });
+  }
+  
+
```

Comment:
@VARTIKASINGH1234 Why you have uploaded node_modules folder in the branch ? repeating mistakes again n again!!